cla_public/static-src/javascripts/modules/bsl_email.js

```diff
@@ -0,0 +1,35 @@
+ 'use strict';
+
+  moj.Modules.BSLEmail = {
+    moreInfos: [],
+
+    init: function() {
+      this.bindEvents();
+    },
+
+    bindEvents: function() {
+      $('#adaptations-bsl_webcam').on('click', function (){
+        moj.Modules.BSLEmail.handleToggle(this);
+      });
+
+      $('#field-adaptations-bsl_email').on('error', function (errors){
+        $('#field-bsl-email-wrapper').removeClass("govuk-inset-text");
+        moj.Modules.BSLEmail.handleToggle($('#adaptations-bsl_webcam')[0]);
+      });
+
+    },
+
+    handleToggle: function(target) {
+      var field_bsl_email_wrapper = $('#field-bsl-email-wrapper');
+      var field_email = $("#field-email")
+      var email = $('#email').val();
+      if (target.checked && email === '') {
```

Comment:
Can this be checked with the Accessibility team as it's out of our usual patterns, or do we already have a pattern that we're following?
I don't think it's straight forward as I understand that back in the day they had some issues with this and that was the original pattern, so wondering if anyone has reservations about our version ( https://accessibility.blog.gov.uk/2021/09/21/an-update-on-the-accessibility-of-conditionally-revealed-questions/ )

You can put a request to the team here: https://digital-accessibility-support-request.form.service.justice.gov.uk/

Reply:
>  or do we already have a pattern that we're following?

@digitalali-moj we are already showing and hiding fields based on a condition. 
On the contact page when you select `Call me back`, the telephone number and other contact fields appear